src/main/java/frc/robot/RobotContainer.java

```diff
@@ -42,16 +42,16 @@
 public class RobotContainer {
   // Joysticks and Operator Input
   public static final Joystick DriverStick = new Joystick(Constants.DriverInputSettings.Driver_Stick_Port);
-  public static final Joystick OperatorStick = new Joystick(Constants.OperatorInputSettings.Operator_Controller_Port);
-  public static final Joystick DSTogglePanel = new Joystick(Constants.DSTogglePanelSettings.DS_Toggle_Panel_Port);
+  //public static final Joystick OperatorStick = new Joystick(Constants.OperatorInputSettings.Operator_Controller_Port);
+  //public static final Joystick DSTogglePanel = new Joystick(Constants.DSTogglePanelSettings.DS_Toggle_Panel_Port);
 
   // Subsystems
   private final Drivetrain drivetrain = new Drivetrain();
-  private final Intake intake = new Intake();
-  private final ROS ros = new ROS();
-  private final Shooter shooter = new Shooter();
-  private final Climber climber = new Climber();
-  private final Solver solver = new Solver();
+  //private final Intake intake = new Intake();
+  //private final ROS ros = new ROS();
+  //private final Shooter shooter = new Shooter();
+  //private final Climber climber = new Climber();
+  //private final Solver solver = new Solver();
```

Comment:
ROS Goodness needs to be re-implemented Soon™

Reply:
It will be done Soon™